views/Navbar.tsx

```diff
@@ -109,7 +109,7 @@ const Navbar = (props: { transparent: boolean }) => {
 				</li>
 			</ul>
 			{/* Mobile Button */}
-			<div onClick={changeNavbar}
+			<div onClick={changeNavbar} role='button'
 				className={
 					'block sm:hidden z-10' + (scrollDown || !props.transparent ? ' text-secondary-dark' : ' text-primary-light')
 				}
```

Comment:
Can I use an <input> element here instead of the div?

Reply:
Who you asking? What difference will an element make?